https://github.com/elastic/search-labs-elastic-co/issues/1454

nehasaini1608 opened the pull request.

JessicaGarson commented:
Thanks, @nehasaini1608, the code runs pretty well here but I did notice a few things here:

1. Could you add a README file here?
2. Also I noticed some `.gitignore` files and `.codespellignore` that you may want to remove from this repo.

nehasaini1608 commented:
@JessicaGarson Thanks for your support. I have removed both .gitignore files and .codespellignore file and updated README . This is done in branch name - 1454
Ready for your review.

JessicaGarson commented:
@nehasaini1608, I had a chance to review feel free to merge as you see fit.
